packages/openapi-generator/src/file-serializer/schema.ts

```diff
@@ -0,0 +1,103 @@
+import { codeBlock } from '@sap-cloud-sdk/util';
+import {
+  OpenApiSchema,
+  OpenApiObjectSchema,
+  OpenApiObjectSchemaProperty
+} from '../openapi-types';
+import { getType } from '../parser/type-mapping';
+import {
+  isReferenceObject,
+  parseTypeNameFromRef,
+  isArraySchema,
+  isObjectSchema,
+  isEnumSchema,
+  isOneOfSchema,
+  isAllOfSchema,
+  isAnyOfSchema,
+  isNotSchema
+} from '../model';
+
+/**
+ * Serialize a schema.
+ * @param schema Parsed schema to be serialized.
+ * @returns The serialized schema
+ */
+export function serializeSchema(schema: OpenApiSchema): string {
```

Comment:
I think the name could be improved. This transforms to string true so it is serializing the object to a format which can be stored in a file. However it is also serializing to TypeScript format so I would like to see this mentioned.

Reply:
As discussed, will be considered for a followup.